midealocal/devices/x0008db/lua_converter.py

```
@@ -0,0 +1,2424 @@
+        elif lua_table.get(self.keyTable["KEY_PROGRAM"]) == "super_concentrated_soak":
+            self.proTable["program"] = 0x91
+        elif lua_table.get(self.keyTable["KEY_PROGRAM"]) == "no_rinse_and_delicate":
+            self.proTable["program"] = 0x92
+        elif lua_table.get(self.keyTable["KEY_PROGRAM"]) == "t_shirt_recovery_pro":
+            self.proTable["program"] = 0x93
+        elif lua_table.get(self.keyTable["KEY_PROGRAM"]) == "customize":
+            self.proTable["program"] = 0xA0
+        
+        # Reservation and time settings
+        if lua_table.get(self.keyTable["KEY_RESERVATION_HOUR"]) is not None:
+            self.proTable["reservationHour"] = string_to_int(lua_table.get(self.keyTable["KEY_RESERVATION_HOUR"]))
```

Ruff (0.16.0)

[error] Undefined name `string_to_int`

(F821)